crd/fake/crd.go

```diff
@@ -0,0 +1,26 @@
+package fake
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/operatorkit/crd"
+)
+
+func MustNewCRD() *crd.CRD {
```

Comment:
All the fake CRD magic is for testing. It helps with the CRD/CRO details that we want to ensure in our tests.